[chore] Update connect-google-cloud-platform-services-new-relic.mdx (#16469)
newrelic/docs-website: src/content/docs/infrastructure/google-cloud-platform-integrations/get-started/connect-google-cloud-platform-services-new-relic.mdx

## Connect GCP to New Relic infrastructure monitoring [#connect]

<Callout variant="important">

>   If this is your first time setting up GCP with New Relic you will need to make sure that Cloud Monitoring API has been enabled for your project which you can do so [here](https://console.cloud.google.com/apis/api/monitoring.googleapis.com/)

Reviewer: ```suggestion
  If this is your first time setting up GCP with New Relic, you need to make sure that you've enabled Cloud Monitoring API for your project. You can do this in the [Google Cloud console](https://console.cloud.google.com/apis/api/monitoring.googleapis.com/).
```

Author: @rhetoric101 it does! Thank you for the edits :)
